opacus/privacy_engine.py

```diff
@@ -26,11 +26,12 @@ def forbid_accumulation_hook(module: nn.Module, _):
 
     for p in module.parameters():
         if hasattr(p, "grad_sample"):
+            pass
             # TODO: this correspond to either not calling optimizer.step()
             # or not calling zero_grad(). Customize message
-            raise ValueError(
-                "Poisson sampling is not compatible with grad accumulation"
-            )
+            # raise ValueError(
+            #     "Poisson sampling is not compatible with grad accumulation"
+            # )
```

Comment:
🦖

Reply:
Not ready, need to fix this. If uncommented, this exception gets raised for some reason.